environment-dev.yml

```
@@ -26,13 +26,13 @@ dependencies:
   - pandas >=2.2
   - parse
   - pyyaml
-  - rechunker
+  - rechunker >=0.5.3
   - scipy >=1.10
```

```suggestion
  - scipy >=1.11.0
```
May as well raise this too.